python/ray/serve/_private/logging_utils.py

```diff
@@ -279,6 +279,7 @@ def configure_component_logger(
     component_type: Optional[ServeComponentType] = None,
     max_bytes: Optional[int] = None,
     backup_count: Optional[int] = None,
+    default_serve_logger: bool = False,
```

Comment:
maybe say `for_default_logger`? otherwise, it feels that this is a logger rather than a flag.